ai4good/params/param_store.py

```diff
@@ -64,7 +64,7 @@ def get_generated_disease_param_vectors(self) -> pd.DataFrame:
 class SimpleParamStore(ParamStore):
 
     def get_models(self) -> List[str]:
-        return ['compartmental-model']
+        return ['compartmental-model','Network-Model','Agent-Based Model']
```

Comment:
can you make them lower case like the first one?

Reply:
agent-based-model, network-model

Reply:
sure

Reply:
As per suggestions changes are done .

Reply:
thanks